postgres/problems/problem_04.md

```
@@ -1,20 +1,23 @@
+* DISTINCT を使用すること
+* WHERE, ORDER BY, JOIN などの追加句は使用しないこと
 
 ## 想定出力例（先頭 5 行）
-
-| user_id | user_name | order_count |
-|---------|-----------|-------------|
-|     101 | Alice     |          37 |
-|     212 | Bob       |          35 |
-| …       | …         |         …  |
+ 
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)